app/assets/stylesheets/components.scss

```diff
@@ -368,9 +368,3 @@ span.showHide-open-all {
     }
   }
 }
-
-
-// ==== not a component ========================================================
-.dgu-unavailable {
```

Comment:
Why removed?

Reply:
We were using `dgu-unavailable` for things that were not ... `unavailable`, so I renamed this to the more general `dgu-secondary-text`.

Moved it to the `dgu.scss` file because it's not a component.

Reply:
ok. We'll have to make it a component and try and use more semantic naming. But I'll take care of that later.